backend/src/main/java/com/google/sps/eventsender/EventSender.java

```diff
@@ -0,0 +1,166 @@
+public class EventSender {
+
+  private static final String APPLICATION_NAME = "Ad-Lib";
+  private static final String TOKENS_DIRECTORY_PATH = "tokens";
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+  /**
+   * Global instance of the scopes required by this quickstart. If modifying these scopes, delete
+   * your previously saved tokens/ folder.
+   */
+  private static final List<String> SCOPES = Collections.singletonList(CalendarScopes.CALENDAR);
+
+  private static final String CREDENTIALS_FILE_PATH = "/credentials.json";
+  private final Calendar service;
+
+  /** @param service Calendar service dependency. */
+  public EventSender(Calendar service) {
+    this.service = service;
+  }
+
+  /**
+   * TODO(): Convert from zonedTimeZone to DateTime
+   *
+   * @param meetingStartTime Time in which the meeting is designated to start at in the format
+   *     "yyyy-mm-dd"
+   * @param timezone String representing the Users Time Zone formatted as an IANA Time Zone Database
+   *     name, e.g. "Europe/Zurich".
+   * @return An EventDateTime with the input meeting time and input time zone.
+   */
+  public EventDateTime setMeetingStartTime(DateTime meetingStartTime, String timezone) {
+    return new EventDateTime().setDateTime(meetingStartTime).setTimeZone(timezone);
+  }
+
+  /**
+   * TODO(): Convert from zonedTimeZone to DateTime
+   *
+   * @param meetingEndTime Time in which the meeting is designated to end at in the format
+   *     "yyyy-mm-dd"
+   * @param timezone String representing the Users Time Zone formatted as an IANA Time Zone Database
+   *     name, e.g. "Europe/Zurich".
+   * @return An EventDateTime with the input meeting time and input time zone.
+   */
+  public EventDateTime setMeetingEndTime(DateTime meetingEndTime, String timezone) {
+    return new EventDateTime().setDateTime(meetingEndTime).setTimeZone(timezone);
+  }
+
+  /**
+   * @param participant1 Instance of the participant class representing an user in a successful
+   *     match
+   * @param participant2 Instance of a participant class representing a user in a successful match
+   * @return An array of EventAttendees with both of the participants emails pre set.
+   */
+  public EventAttendee[] createParticipantsArray(
+      com.google.sps.data.Participant participant1, com.google.sps.data.Participant participant2) {
+    return new EventAttendee[] {
+      new EventAttendee().setEmail(participant1.getUsername() + "@google.com"),
+      new EventAttendee().setEmail(participant2.getUsername() + "@google.com"),
+    };
+  }
+
+  /**
+   * Creates a notification trigger within the event so that the user will receive an email reminder
+   * in 5 minutes as wel as a push notification to their screen in 1 minute
+   *
+   * @return Returns these options of notifications to be passed into the event as parameters.
+   */
+  public Event.Reminders setReminderNotifications() {
```

Comment:
IIUC, the user of this class can just instantiate an object and call createEvent() with the arguments, right? If so, I think there's no need to make the set* functions as public?